Medinite compliance on Inhuman Strength

Turtlelord26 opened the pull request:
#773 
Slightly buffed, now interacts favorably with shield/cloaking modules.

Now makes no sense within its own branch, but infinitely more with the nanite removal branch factored in.

@dfuchs2 Rewrote your trait, review plox.
@rosebrallier Playing with the skill economy, review plox.

Turtlelord26 committed:
Medinite compliance on Inhuman Strength

Slightly buffed, now interacts favorably with shield/cloaking modules.

Now makes no sense within its own branch, but infinitely more with the nanite removal branch factored in.

dfuchs2 reviewed (approved):
Looks good to me

rosebrallier commented on CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:
```diff
@@ -100,11 +100,11 @@ specialty drone armor sets.
 
 Inhuman Strength:
     Built specifically for lifting and holing great weights, you are far 
```
While we're here, can we creep out to fix "holing"?

rosebrallier reviewed (commented):
I approve this message!

Turtlelord26 committed:
Typography

Unless we meant for them to be really good with a very specific kind of power tool attachment 🐢
